src/Kiota.Builder/Resources/AdaptiveCardTemplate.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "type": "AdaptiveCard",
+  "version": "1.0",
```

Comment:
Not a blocking question, but why are we not using the latest version of AdaptiveCard

Reply:
I'll be honest, I didn't counter-check the version. I used the template defined in the issue description https://github.com/microsoft/kiota/issues/5674